pre-docs/bolt_installing.md

```diff
@@ -201,6 +201,12 @@ If Ruby is already included in your operating system, verify that it is version
     ```
     bolt --help
     ```
+4. To install bolt's Puppet module dependencies:
+
+    ```
+    cd bolt
+    bolt puppetfile install
```

Comment:
I'm not sure this will work. Bolt's puppetfile includes an absolute module path. Although from my testing we may be ignoring that.

Reply:
Oh, this uses `~/.puppetlabs/bolt/Puppetfile`. And I don't see any way to override it. So I think the instructions need to say: add the modules in https://github.com/puppetlabs/bolt/blob/master/Puppetfile to `~/.puppetlabs/bolt/Puppetfile`.